### DISCUSSION TITLE
Vercel domain contact privacy

### DISCUSSION THREAD

**Discussion by ruchernchong** [vercel/community#1435] [Help] [answered]
### Summary

I am not able to find any details on this in the docs. I want to check if the domains bought/transferred to Vercel have privacy protection for the contact details.

**Comment by amyegan** [comment, marked as answer]
That is a great question @ruchernchong. I checked with my team and they confirmed that Vercel's contact info is used for WHOIS and ICANN lookup

**Reply by ruchernchong** [reply]
Awesome! Thank you!
